Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -682,19 +696,26 @@ namespace AtomSampleViewer
                     }
                 }
                 ImGui::SameLine();
-                if (ImGui::Button("Next"))
+                if (ImGui::Button("Add and Next"))
```

Comment:
I don't get the "Add and Next" wording.
BTW, should we force the user to pick one of the radio options before they can proceed? Maybe gray out the button until the do?

Reply:
This was meant to be "Add to final report summary, and go to next screenshot for manual inspection". Let me try making the process more explicit.

Reply:
I've added a text during manual inspection to indicate whether or not the screenshot has been added.